src/main/java/net/imglib2/img/array/ArrayImg.java

```diff
@@ -79,10 +80,11 @@ public ArrayImg( final A data, final long[] dim, final Fraction entitiesPerPixel
 		this.data = data;
 	}
 
+	@SuppressWarnings("unchecked")
 	@Override
 	public A update( final Object o )
 	{
-		return data;
+		return (A) data.createView(o);
```

Comment:
Does not conform with imglib2 formatting. I saw that in a few other places. Probably best to run the formatter on all files.

Reply:
I could use some help with this. How do I access the formatter? Is this just the generic Eclipse formatter or is there a imglib2 format ruleset I'm missing?

Reply:
I typically use the [Eclipse formatter](https://github.com/imglib/imglib2/blob/master/doc/imglib2-eclipse-mars-clean-up-formatter-style.xml). I have not had the best experience with it in IntelliJ but I see there was a [commit to add a formatter for IntelliJ](https://github.com/imglib/imglib2/commit/9998be95f18962c4e8a09f4afdb7502a30f0eded) just three months ago.

Reply:
Resolved and formatted as of b4f1233